pages/content/amp-dev/documentation/guides-and-tutorials/develop/style_and_layout/style_pages.md

```diff
@@ -14,6 +14,12 @@ contributors:
   - choumx
 ---
 
+[filter formats="email"]
+Note: This document is in addition to rules specific to AMP for Email. See
```

Comment:
Suggestion: rephrase `This document is in addition to rules specific to AMP for Email.` to "AMP for Email specifies additional CSS constraint which are described in here: ....`

Reply:
Good point, done.